Classes/Util.php

```diff
@@ -134,6 +137,34 @@ public static function isDraftRecord(
         return $isWorkspaceRecord;
     }
 
+    public static function skipHooksForRecord(
+        string $table,
+        int $uid,
+        ?int $pid
+    ): bool {
```

Comment:
@dkd-kaehm, maybe have some kind of exclude list defined here? `sys_template`, `be_groups` , `be_users`